k8s/scripts/crio-build.sh

```diff
@@ -9,7 +9,7 @@
 # Usage: docker run -v $(shell pwd)/bin:/mnt/results crio-bld
 #
 
-declare -a CRIO_VERSIONS=(v1.24 v1.25 v1.26 v1.27)
+declare -a CRIO_VERSIONS=(v1.26 v1.27 v1.28)
```

Comment:
I believe 1.25 is still supported by various cloud providers, so I would leave it for now.

Reply:
It's no longer supported on GKE, and AKS will stop supporting it in a [few days (Jan 14)](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar).

Plus it would mean we have to support 4 K8s releases which is hard.